fhir-persistence/src/main/java/com/ibm/fhir/persistence/context/impl/FHIRPersistenceContextImpl.java

```diff
@@ -20,7 +20,7 @@
     private FHIRPersistenceEvent persistenceEvent;
     private FHIRHistoryContext historyContext;
     private FHIRSearchContext searchContext;
-    private boolean includeDeleted;
+    private boolean includeDeleted = false;
```

Comment:
Already defaults to false, but doesn't hurt to make sure.

Reply:
yeah, I didn't actually mean to commit this file...I added that locally a while back so that I would remember what the default value is.  I guess its fine though.